index.html

```diff
@@ -81,6 +81,51 @@
         </div>
       </div>
 
+      <div class="col-xs-12 fave-images">
```

Comment:
The favorite images section looks good, and the addition of the "favorite" and "Delete" buttons are a nice touch.